### PR TITLE
Make error messages consistent

### PR DESCRIPTION
**Pull request by tonglil**

### CODE REVIEW

File: drone/deploy/deploy.go [hunk 1]
```diff
@@ -96,7 +96,7 @@ func deploy(c *cli.Context) error {
 
 	env := c.Args().Get(2)
 	if env == "" {
-		return fmt.Errorf("Please specify the target environment (ie production)")
+		return fmt.Errorf("Error: Please specify the target environment (e.g. production).")
```
Comment (tboerger) [inline drone/deploy/deploy.go]: Nearly all messages are errors, so why do we highlight it here?

Comment (tonglil) [reply drone/deploy/deploy.go]: I'm not sure, they are highlighted here: https://github.com/drone/drone-cli/blob/master/drone/internal/util.go#L31

Comment (tonglil) [reply drone/deploy/deploy.go]: Can we agree on a certain format (either remove all `Error:`s or add it)?

Comment (tboerger) [reply drone/deploy/deploy.go]: I would suggest to remove it

**Comment by bradrydzewski** [comment]
> Can we agree on a certain format (either remove all Error:s or add it)?

Yes, we definitely should.

I would suggest we pick a tool that Drone developers are likely familiar with (git, docker) and we should adopt their convention.

Docker error format:

```
Error: No such object: xxx
```

Git error format:

```
fatal: Not a git repository (or any of the parent directories): .git
```

**Comment by tonglil** [comment]
Since we have `Error:` already, let's go with Docker error format for CLI error outputs.

**Comment by bradrydzewski** [comment]
hey sorry for delayer here

**Pull request merged**